pkg/relay/README.md

```diff
@@ -0,0 +1,81 @@
+# Relay
+
+`ion-sfu` supports relaying tracks to other ion-SFUs or other services using the ORTC API.
+
+Using this api allows to quickly send the stream to other services by signaling a single request, after that all the following negotiations are handled internally.
+
+## API
+
+### Relay Peer
+
+The relay peer shares common methods with the Webrtc PeerConnection, so it should be straight forward to use. To create a new relay peer follow below example
+:
+```go
+ // Meta holds all the related information of the peer you want to relay.
+ meta := PeerMeta{
+	      PeerID : "super-villain-1",
+	      SessionID : "world-domination",
+        } 
+ // config will hold pion/webrtc related structs required for the connection. 
+ // you should fill according your requirements or leave the defaults.
+ config := &PeerConfig{} 
+ peer, err := NewPeer(meta, config)
+ handleErr(err)
+ 
+ // Now before working with the peer you need to signal the peer to 
+ // your remote sever, the signaling can be whatever method you want (gRPC, RESt, pubsub, etc..)
+ signalFunc= func (meta PeerMeta, signal []byte) ([]byte, error){
+   if meta.session== "world-domination"{
+      	return RelayToLegionOfDoom(meta, signal)
+   }	
+   return nil, errors.New("not supported")
+ }
+ 
+ // The remote peer should create a new Relay Peer with the metadata and call Answer.
+ if err:= peer.Offer(signalFunc); err!=nil{
+ 	handleErr(err)
+ }
```

Comment:
This above usage is wrapped up in [Publisher's Relay](https://github.com/pion/ion-sfu/blob/feat/relayingest/pkg/sfu/publisher.go#L195) method correct?